### PR TITLE
[WFCORE-4309] Value validator for 'host-context-map' attribute of 'se…

### PR DESCRIPTION
**Pull request by Skyllarr**
…rver-ssl-sni-context' resource

JIRA: https://issues.jboss.org/browse/WFCORE-4309

### CODE REVIEW

File: elytron/src/main/java/org/wildfly/extension/elytron/_private/ElytronSubsystemMessages.java [hunk 1]
```diff
@@ -557,4 +557,6 @@
     @Message(id = 1060, value = "Fileless KeyStore needs to have a defined type.")
     OperationFailedException filelessKeyStoreMissingType();
 
+    @Message(id = 1061, value = "Value of host context map contains invalid hostname.")
```
Comment (bstansberry) [inline elytron/src/main/java/org/wildfly/extension/elytron/_private/ElytronSubsystemMessages.java]: This message should include the invalid value.

Comment (Skyllarr) [reply elytron/src/main/java/org/wildfly/extension/elytron/_private/ElytronSubsystemMessages.java]: @bstansberry Thank you. Fixed.

**Comment by Skyllarr** [comment]
@darranl I have incorporated the changes we talked about. I added precompiled pattern and removed the toLowerCase method (was slower) and added the licence header to the test file.

**Comment by darranl** [comment]
@bstansberry do you have any more review on this one?  Just trying to gauge if this can come in as a bug or if additional validation could suggest RFE?

**Pull request merged**